Src/AutoNSubstitute/AutoNSubstitute.csproj

```diff
@@ -64,6 +64,7 @@
   <ItemGroup>
     <Compile Include="AutoConfiguredNSubstituteCustomization.cs" />
     <Compile Include="AutoNSubstituteCustomization.cs" />
+    <Compile Include="AutoSubstituteCustomization.cs" />

```

Comment:
Formally, it would be a breaking change.

Sometimes we do accept that a few unit tests break (until we update them), if the breaks happen because the tests are a bit too overspecified. It's a judgement call every time, though.

I'd tend to consider changing the type of the `Builder` property a breaking change. Regarding the other change, it'd depend on whether or not it changes the behaviour of the system.

What do you have in mind?